content/archived.txt

```diff
@@ -0,0 +1,16640 @@
+/pl/docs/xpcom
```

Comment:
Can you add a couple of lines to the top of this file that explains what it is and why it exist?
E.g.
```
# This file is ...
# We need this because....

/pl/docs/xpcom
/pl/docs/zmiany_w_obiekcie_xmlhttprequest_w_gecko_1.8
...
```
```